xrpl/utils/txn_parser/utils/balance_parser.py

```diff
@@ -0,0 +1,161 @@
+"""Helper functions for balance parser."""
+
+from decimal import Decimal
+from typing import Callable, List, Optional
+
+from pydash import group_by  # type: ignore
+
+from xrpl.utils.txn_parser.utils.nodes import NormalizedNode
+from xrpl.utils.txn_parser.utils.types import Balance, BalanceChange, BalanceChanges
+from xrpl.utils.xrp_conversions import drops_to_xrp
+
+
+def _get_xrp_quantity(
+    node: NormalizedNode,
+    value_parser: Callable[[NormalizedNode], Optional[Decimal]],
+) -> Optional[BalanceChange]:
+    value = value_parser(node)
+    if value is None:
+        return None
+    value_is_negative = value < Decimal(0)
+    value = drops_to_xrp(str(value.copy_abs()))
+    if value_is_negative:
+        value = Decimal(f"-{value}")
```

Comment:
What's the objective of this?

Reply:
The `drops_to_xrp` function only accepts positive values. This is supposed to negate the value again. I guess I made it a little bit more readable now.